src/app/components/privacy-policy/privacy-policy.component.spec.ts

```diff
@@ -68,4 +79,24 @@ describe('PrivacyPolicyComponent', () => {
     });
   });
 
+  it('should have called the scroll function when scroll selected element is called', () => {
+    const scrollSelected = de.query(By.css('.scroll-selected'));
+    scrollSelected.nativeElement.click();
+
+    fixture.whenStable().then(() => {
+      expect(component.scroll).toHaveBeenCalled();
+      expect(component.highlightNav).toHaveBeenCalled();
+    });
+  });
+
+  it('should have called the highlight functions when scroll selected element is called', () => {
+    const scrollSelected = de.query(By.css('.scroll-selected'));
+    scrollSelected.nativeElement.click();
+
+    fixture.whenStable().then(() => {
+      expect(component.highlightNav).toHaveBeenCalled();
+      expect(component.highlightSectionTitle).toHaveBeenCalled();
```

Comment:
This could have been written along with the above two "Expect"s.

Reply:
@lunayach  done